src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -41,7 +41,6 @@
     <_PackageFiles Remove="**\obj\**\*.cs" />
     <_PackageFiles Remove="Core\**\*.cs" />
     <_PackageFiles Remove="AssemblyInfo.cs" />
-    <_PackageFiles Remove="EndpointTemplates\ExternallyManagedContainerServer.cs" />
```

Comment:
ugh, what is causing this to happen? always trips me up :(

Reply:
Something in the project system sees those entries and thinks it needs to modify them.